powerdns/tests/test_permissions.py

```diff
@@ -199,20 +191,21 @@ def test_user_can_create_records_in_unrestricted_domain(self):
         """Normal user can create records in domain that is marked as
         'unrestricted'."""
         request = self.u_client.post(
-            reverse('api:default:record-list'),
+            reverse('api:v2:record-list'),
             {
                 'name': 'site.u.example.com',
                 'content': '192.168.1.4',
                 'type': 'A',
-                'domain': get_domain_url(self.unrestricted_domain),
+                'domain': self.unrestricted_domain.id,
+                'service': self.service.id,
             },
         )
         self.assertEqual(request.status_code, 201)
 
     def test_user_cant_create_records_in_other_domains(self):
```

Comment:
Great you've noticed this. 400 is returned because of api v2 demands domain PK and not URL, what is more also service pk is also required. Now it is fixed.